### PR TITLE
fix(lint): Make sure no console log rule works

### PR DESCRIPTION
**Pull request by kevinuma**

### CODE REVIEW

File: scripts/constructEmergencyRoot.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-process-exit */
+/* eslint-disable no-console */
```
Comment (kevinuma) [inline scripts/constructEmergencyRoot.ts:2]: Script so okay to leave console.logs here.

File: scripts/sendTokens.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-process-exit */
+/* eslint-disable no-console */
```
Comment (kevinuma) [inline scripts/sendTokens.ts:2]: ditto

File: scripts/unwrapWeth.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-process-exit */
+/* eslint-disable no-console */
```
Comment (kevinuma) [inline scripts/unwrapWeth.ts:2]: ditto

File: src/utils/GckmsUtils.ts [hunk 4]
```diff
@@ -35,6 +35,7 @@ export function getGckmsConfig(keys) {
     try {
       if (fs.existsSync(`${__dirname}/${overrideFname}`)) configOverride = require(`./${overrideFname}`);
     } catch (err) {
+      // eslint-disable-next-line no-console
```
Comment (kevinuma) [inline src/utils/GckmsUtils.ts:38]: Not sure what this is used for yet + no access to logger here so I'll leave this console.log here for now

File: src/utils/Help.ts [hunk 5]
```diff
@@ -11,6 +11,7 @@ export function usage(badInput: string = undefined) {
     \tts-node ./index.tx [-h] <--dataworker|--finalizer> --wallet <${walletOpts}>
   `.slice(1); // Skip leading newline
 
+  // eslint-disable-next-line no-console
```
Comment (kevinuma) [inline src/utils/Help.ts:14]: Not removed for the same reason as above

File: src/utils/Help.ts [hunk 6]
```diff
@@ -56,6 +57,7 @@ export function help() {
     \tRelayer Instructions: ${relayerDocsUrl}
   `.slice(0, -1); // Skip trailing newline
 
+  // eslint-disable-next-line no-console
```
Comment (kevinuma) [inline src/utils/Help.ts:60]: Not removed for the same reason as above

**Comment by kevinuma** [comment]
> You should look into overriding `eslint` rules for particular paths [here](https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-based-on-glob-patterns) rather than ignore on a file by file basis.

I originally didn't want to skip all rules for specific dirs and it's not easy to only skip specific rules for certain dirs without creating more eslint config files in local dirs.

**Review by nicholaspai — APPROVED** [review]
Nice LGTM! The `overrides` rule will be very useful in `/scripts` and I agree with not touching the `utils/` files as they are potentially used all in unknown ways

**Pull request merged**